modules/submitBook.js

```diff
@@ -0,0 +1,18 @@
+import AwesomeBook from './awesomeook.js';
+
+function submitBook() {
+  const form = document.querySelector('form');
+  const awesomeBook = new AwesomeBook();
+  form.addEventListener('submit', (e) => {
+    e.preventDefault();
+    const [titleInput, authorInput] = e.target.elements;
+    awesomeBook.insert({
+      author: authorInput.value,
+      title: titleInput.value,
+    });
+    e.target.reset();
+  });
+}
+
+// eslint-disable-next-line import/prefer-default-export
+export { submitBook };
```

Comment:
- [ ] Please, It's one of the project requirements you use `arrow` syntax for functions instead of JavaScript Function Expressions because in this project you learn how to use ES6 new features. Kindly make this :wink: 💯 

- For more documentation, you can visit this [link](https://www.w3docs.com/learn-javascript/function-expressions.html)

- [ ] I suggest you develop the practice of not disabling linters in your codebase, but rather research for solutions that will help you solve the linter issues because linters are set up to ensure you write clean and bug-free code. 😉 👍🏻

Reply:
Done, thanks